modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1676,7 +1676,7 @@
     <string name="onboarding_import_from_castbox_step_2">Tap the \“Personal\” tab.</string>
     <string name="onboarding_import_from_castbox_step_3">Tap “Settings”.</string>
     <string name="onboarding_import_from_castbox_step_4">Scroll down to \“OPML Export\” and tap it.</string>
-    <string name="onboarding_import_from_castbox_step_5">@string/onboarding_import_from_other_apps_step_2</string>
+    <string name="onboarding_import_from_castbox_step_5" translatable="false">@string/onboarding_import_from_other_apps_step_2</string>
```

Comment:
<img width="987" alt="Screenshot 2023-07-31 at 11 16 46 pm" src="https://github.com/Automattic/pocket-casts-android/assets/308331/cd8e82e6-d387-4ec3-86a7-5cbfd3920220">

This is coming through to be translated.